.github/workflows/backport-pr-manual.yml

```diff
@@ -0,0 +1,171 @@
+name: 'Manually Cherry-Pick to Release Branches'
+
+on:
+  workflow_dispatch:
+    inputs:
+      merge_commit_sha:
+        description: 'The sha of the merge commit from the main PR.'
+        required: true
+
+env :
+  TERRAFORM_MAINTAINERS: ${{ vars.TERRAFORM_MAINTAINERS }} # eg. '["matttrach"]'
+
+jobs:
+  create-cherry-pick-prs:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+      issues: write
+      actions: write
+    steps:
+      - name: 'Wait for merge to settle'
+        uses: actions/github-script@ed597411d8f924073f98dfc5c65a23a2325f34cd # v8.0.0 https://github.com/actions/github-script
+        env:
+          TERRAFORM_MAINTAINERS: ${{ env.TERRAFORM_MAINTAINERS }}
+        with:
+          script: |
+            const owner = context.repo.owner;
+            const repo = context.repo.repo;
+            const mergeCommitSha = context.payload.head_commit.id;
```

Comment:
`context.payload.head_commit.id` is only available on push events, not workflow_dispatch events.
ref: 
- https://docs.github.com/en/webhooks/webhook-events-and-payloads#:~:text=the%20ref.-,head_commit,-object%20or%20null

Reply:
Good catch! Fixed